tests/framework/ensembleModelTests/test_ensemble_model_2_codes_optional_output.xml

```diff
@@ -6,14 +6,15 @@
     <created>2017-05-17</created>
     <classesTested>Models.Code, JobHandler.Thread</classesTested>
     <description>
-       The main goal of this test is to show how to specify optional output in the Ensemle Model object.
+       The main goal of this test is to show how to specify optional output in the Ensemble Model object.
+       All the RAVEN available data storage types are tested (DataObjects and Databases).
     </description>
```

Comment:
Git does track when these files are revised and what the specific changes made were. I guess I am on the fence of whether this needs to be tracked so vigorously. I don't want to this to become extra work for the sake of extra work. If the case can be made as to why adding the revision tag is better than tracing edits in the git history, then I am happy to have it in the test file, otherwise I can see this kind of comment slowing down our progress as we would now have to wait for the whole test system to finish before accepting this PR. Thoughts?